src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -4079,6 +4079,7 @@ namespace OutputReportTabular {
         static int curFirstColumn(0);
 
         if (!DoWeathSim) return;
+        assert(DataGlobals::TimeStepZoneSec > 0.0);
```

Comment:
Other unit tests were calling this function without setting TimeStepZoneSec which initializes to zero. Add an assert here to catch this.